.travis.yml

```diff
@@ -48,7 +48,9 @@ before_script:
 - export INSTALL_PREFIX=~/torch/install
 -  ls $HOME/OpenBlasInstall/lib || (cd /tmp/ && git clone https://github.com/xianyi/OpenBLAS.git -b master && cd OpenBLAS && (make NO_AFFINITY=1 -j$(getconf _NPROCESSORS_ONLN) 2>/dev/null >/dev/null) && make PREFIX=$HOME/OpenBlasInstall install)
 - git clone https://github.com/torch/distro.git ~/torch --recursive
+- sudo -E bash ~/torch/travis_cuda_install.sh
```

Comment:
this would be not ~/torch/travis_cuda_install.sh but
sudo -E bash $ROOT_TRAVIS_DIR/travis_cuda_install.sh